### PR TITLE
nc-news-7

### PR DESCRIPTION
**Pull request by AlinaKhvishchuk**

### CODE REVIEW

File: __tests__/api.test.js [hunk 1]
```diff
@@ -221,3 +221,60 @@ describe("GET /api/articles/:article_id/comments", () => {
...
+    };
+    return request(app)
+      .post("/api/articles/2o/comments")
+      .send(testNewComment)
+      .then((response) => {
+        expect(response.status).toBe(400);
+        expect(response.body.msg).toBe("Bad request");
+      });
+  });
+
```
Comment (pippaamy) [inline __tests__/api.test.js:298]: can you add a test for if the user does not exist!

File: __tests__/api.test.js [hunk 2]
```diff
@@ -221,3 +221,60 @@ describe("GET /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("POST /api/articles/:article_id/comments", () => {
+  it("POST status: 201, responds with the posted comment", () => {
+    const testNewComment = {
+      username: "icellusedkars",
+      body: "Great news!Way to go!",
+    };
```
Comment (pippaamy) [inline __tests__/api.test.js:230]: can we add another 201 test that it will ignore unnecessary properties!

**Review by pippaamy — COMMENTED** [review]
great job, lets add these tests on!

**Review by xViolaine — APPROVED** [review]
Nice implementation of the feedback given to you! :D

**Pull request merged**